drivers/sensor/si7006/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019 Electronut Labs
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SI7006
+	bool "SI7006 Temperature and Humidity Sensor"
```

Comment:
The vendor preferred spelling of this sensor is `Si7006`.  The Kconfig name has to be all caps for style, but this use and use in documentation should be mixed-case.